Runtime/Native/Windows/NativeClient.cs

```diff
@@ -0,0 +1,438 @@
+            {
+                Debug.LogWarning("Backtrace native integration status: database path doesn't exist");
+                return;
+            }
+
+            var crashpadHandlerPath = GetDefaultPathToCrashpadHandler();
+            if (!File.Exists(crashpadHandlerPath))
+            {
+                Debug.LogWarning("Backtrace native integration status: Cannot find path to Crashpad handler.");
+                return;
+            }
+
+            var minidumpUrl = new BacktraceCredentials(_configuration.GetValidServerUrl()).GetMinidumpSubmissionUrl().ToString();
+
+            if (!Directory.Exists(databasePath))
+            {
+                Directory.CreateDirectory(databasePath);
+            }
+
+            // reassign to captureNativeCrashes
+            // to avoid doing anything on crashpad binary, when crashpad isn't available
+            _captureNativeCrashes = Initialize(
+                minidumpUrl,
+                databasePath,
+                crashpadHandlerPath,
+                attachments.ToArray(),
+                attachments.Count());
+
+            if (!_captureNativeCrashes)
+            {
+                Debug.LogWarning("Backtrace native integration status: Cannot initialize Crashpad client");
+                return;
+            }
+
+            foreach (var attribute in clientAttributes)
+            {
+                AddNativeAttribute(attribute.Key, attribute.Value);
+            }
+
+            // add exception type to crashes handled by crashpad - all exception handled by crashpad 
+            // by default we setting this option here, to set error.type when unexpected crash happen (so attribute will present)
+            // otherwise in other methods - ANR detection, OOM handler, we're overriding it and setting it back to "crash"
+
+            // warning 
+            // don't add attributes that can change over the time to initialization method attributes. Crashpad will prevent from 
+            // overriding them on game runtime. ANRs/OOMs methods can override error.type attribute, so we shouldn't pass error.type 
+            // attribute via attributes parameters.
+            AddNativeAttribute("error.type", "Crash");
+        }
+
+        public void Disable()
+        {
+            if (_anrThread != null)
+            {
+                _stopAnr = true;
+            }
+            return;
+        }
+
+        public void GetAttributes(IDictionary<string, string> attributes)
+        {
+            return;
+        }
+
+        public void HandleAnr(string gameObjectName = "", string callbackName = "")
+        {
+            if (!_captureNativeCrashes || !_configuration.HandleANR)
+            {
+                return;
+            }
+
+            bool reported = false;
+            var mainThreadId = Thread.CurrentThread.ManagedThreadId;
+            _anrThread = new Thread(() =>
```

Comment:
I recognize this code, sorry for being lazy but is it duplicated or just moved?

Reply:
we have dedicated ANR thread per native integration. it's the same integration that we have in other native clients and we should move our integration to generic class, however it's out of scope and generic implementation might break stuff on Android/iOS without additional tests.

Reply:
Ok. Can you please add it to technical debt for future work?